enhancements/installer/alibaba-cloud-ipi.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Alibaba Cloud IPI
+authors:
+  - "@lemondlut"
+reviewers:
+  - "@bradteng"
+  - "@rosawy"
+approvers:
+  - "@?"
+creation-date: 2021-03-25
+last-updated: 2021-03-25
+status: implementable
+
+---
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement covers adding support to install OpenShift clusters to Alibaba Cloud and details for the Installer to create infrastructure and run an OpenShift cluster on Alibaba Cloud.
+
+## Motivation
+
+Currently, openshift4.x does not support Alibaba Cloud via IPI installation. However, the feedback team got is that there is a lot of demand for this kind of installation.
+
+### Goals
+
+1. Installer creates infrastructure and installs OpenShift on Alibaba Cloud.
+
+### Non-Goals
+
+## Proposal
+
+### User Stories
+Enables `openshift-install` to create and destroy a cluster on Alibaba Cloud, which is managed and released and covered by CI tests.
+
+### Implementation Details/Notes/Constraints [optional]
+
+
+### Risks and Mitigations
```

Comment:
Should users expect differences in the resources and their available options across different regions, and if so, how will we handle the differences? For example, can users expect the same level of performance across all regions?